scripts/test-setup.sh

```diff
@@ -12,6 +12,23 @@ CONT_NAME="${OS_TYPE}-${OS_VERSION}-${COMPONENT}"
 CERTGEN_REPO="https://github.com/redhat-qe-security/certgen"
 CERTGEN_PATH="openssl/Library/certgen"
 
+# Test sanity check
+# Check if all tests have rlGetTestState at their end
+FAILED_CHECKS=0
+FAILED_NAMES=()
+while read file; do
+    if ! grep -Pzq "rlGetTestState[[:space:]]*$" "$file"; then
```

Comment:
No, `-z` replaces all newlines with a NUL character (`\0`), so the `$` anchor in this case matches the EOF.